roles/spark/templates/spark-defaults.conf.j2

```diff
@@ -19,10 +19,17 @@
 # This is useful for setting default environmental settings.
 
 # Example:
-spark.master                     mesos://zk://master01:2181,master02:2181,master03:2181/mesos
-# spark.eventLog.enabled           true
-# spark.eventLog.dir               hdfs://namenode:8021/directory
-# spark.serializer                 org.apache.spark.serializer.KryoSerializer
+
+spark.master                     {{mesos_master}}
+
+spark.eventLog.enabled           true
+spark.eventLog.dir               hdfs:///shared/spark_logs
+
+spark.history.fs.logDirectory    hdfs:///shared/spark_logs
+
+# spark.serializer               org.apache.spark.serializer.KryoSerializer
+
 spark.driver.memory              5g
 spark.executor.extraJavaOptions  -XX:+PrintGCDetails -Dkey=value -Dnumbers="one two three"
-spark.mesos.executor.home	 /usr/lib/spark/spark-1.6.0-bin-hadoop2.6
+
+spark.mesos.executor.home	       {{spark_home}}
```

Comment:
Yes, it will still use the derby but is it still creating the warehouse folder in the local file system? If it create the warehouse folder in HDFS instead on the local, it's already better, no..? I will search for the documentation and send it to you though..